index.js

```diff
@@ -36,6 +36,9 @@ function formatName (arr) {
 }
 
 function processString (str, system) {
+  if (!str) {
+    throw new Error('Input contains non-Chinese characters');
```

Comment:
Oops, sorry about missing this earlier – could you remove that `;`? This project uses [standard](https://github.com/feross/standard#rules). :sparkles: